packages/block-editor/src/components/rich-text/index.js

```diff
@@ -65,6 +65,7 @@ class RichTextWrapper extends Component {
 		this.onPaste = this.onPaste.bind( this );
 		this.onDelete = this.onDelete.bind( this );
 		this.inputRule = this.inputRule.bind( this );
+		this.getAllowedFormats.EMPTY_ARRAY = [];
```

Comment:
Why not just a module constant

Reply:
This empty array should be used specifically for the allowed formats state. I don't see a reason to make it a module constant. It isn't and shouldn't be reused for other state.

Reply:
Would having a more declarative name be of any help? E.g. `this.getAllowedFormats.NO_FORMATS = []`

Reply:
I can also use a module variable and name it differently. I just have this habit of putting stuff close to what is using it.